Services/BloodBoundService.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections.Generic;
+using ProjectM;
+using Stunlock.Core;
+using Unity.Entities;
+
+namespace KindredCommands.Services;
+
+/// <summary>
+/// Manages blood-bound category.
+/// </summary>
+public class BloodBoundService
+{
+	/// <summary>
+	/// Initializes a new instance of <see cref="BloodBoundService"/>.
+	/// </summary>
+	public BloodBoundService()
+	{
+		SetBloodBound(Core.ConfigSettings.BloodBound);
+	}
+
+	/// <summary>
+	/// Updates category for every entity in <paramref name="bloodBound"/>.
+	/// </summary>
+	/// <param name="bloodBound">Key: prefab guid, Value: indicates if entity belongs to blood-bound category.</param>
+	public void SetBloodBound(IReadOnlyDictionary<string, bool> bloodBound)
```

Comment:
Instead of this being a third override of SetBloodBound would be nice if it was named like SetBloodBoundAtStartup for self documentation.  At first glance, it wasn't clear this wasn't the one called by the command but by the constructor.  Also it shouldn't be public as we don't want anyone else to call it.

```suggestion
	void SetBloodBoundAtStartup(IReadOnlyDictionary<string, bool> bloodBound)
```